sdf/1.9/inertial.sdf

```diff
@@ -27,24 +38,54 @@
   </element>
 
   <element name="inertia" required="0">
-    <description>The 3x3 rotational inertia matrix. Because the rotational inertia matrix is symmetric, only 6 above-diagonal elements of this matrix are specified here, using the attributes ixx, ixy, ixz, iyy, iyz, izz.</description>
+    <description>
+      This link’s moments of inertia ixx, iyy, izz and products of inertia
+      ixy, ixz, iyz about Co (the link’s center of mass) for the unit vectors
+      Ĉx, Ĉy, Ĉᴢ fixed in the center-of-mass-frame C.
+      Note: the orientation of Ĉx, Ĉy, Ĉᴢ relative to L̂x, L̂y, L̂ᴢ is specified
+      by the `pose` tag.
+      To avoid compatibility issues associated with the negative sign
+      convention for product of inertia, align Ĉx, Ĉy, Ĉᴢ with principal
+      inertia directions so that all the products of inertia are zero.
+      For more information about this sign convention, see the following
+      Mathworks documentation for working with CAD tools:
+      https://www.mathworks.com/help/releases/R2021b/physmod/sm/ug/specify-custom-inertia.html#mw_b043ec69-835b-4ca9-8769-af2e6f1b190c
+    </description>
     <element name="ixx" type="double" default="1.0" required="1">
-      <description></description>
+      <description>
+        The link’s moment of inertia about Co (the link’s center of mass) for Ĉx.
+      </description>
     </element>
     <element name="ixy" type="double" default="0.0" required="1">
-      <description></description>
+      <description>
+        The link’s product of inertia about Co (the link’s center of mass) for
```

Comment:
nit (here and elsewhere) `link’s` => `link's` (different character?)

Reply:
https://github.com/ignitionrobotics/sdformat/pull/990/commits/77630ddd7248f55178cc92acda9cbc0391eaf9d9